src/DataFrames.jl

```diff
@@ -12,7 +12,8 @@ using Compat
 import Compat.String
 using Reexport
 @reexport using StatsBase
-@reexport using DataArrays
+@reexport using NullableArrays
```

Comment:
I think the idea was that people shouldn't have to call `using` to be able to use methods from these packages, since one can hardly work with DataFrames without also loading these packages.